reactful-numbers/src/app/components/Button.js

```diff
@@ -1,5 +1,5 @@
 import React from "react";
-function Button(buttonName) {
-  return <button className="button">Button</button>;
+function Button({ buttonColor, buttonText }) {
```

Comment:
Passed props from parent to child component to style specific buttons differently